x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -84,7 +84,6 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
         icon: APP_ICON,
         path: APP_OVERVIEW_PATH,
         order: 300,
-        className: 'homSolutionsSection__blackCard',
```

Comment:
See comment above regarding why we likely need to keep this (or something similar to it).

Reply:
See my comment above as well :)